.github/PULL_REQUEST_TEMPLATE/develop-template.md

```diff
@@ -0,0 +1,18 @@
+Originator(s):
+
+Description (include issue title and the keyword ['closes', 'fixes', 'resolves'] and issue number):
+
+List all namelist files that were added or changed:
+
+List all files eliminated and why:
+
+List all files added and what they do:
+
+List all existing files that have been modified, and describe the changes: 
+(Helpful git command: `git diff --name-status development...<your_branch_name>`)
+
+List any test failures:
+
+Is this an answer-changing PR? If so, is it a new physics package, algorithm change, tuning change, etc?
+
+If yes to the above question, list how this code was validated with the new/modified features?
```

Comment:
I might change `list` to `describe` here, and get rid of the question mark:

```suggestion
If yes to the above question, describe how this code was validated with the new/modified features:
```

Reply:
changed!